tests/contrib/botocore/test.py

```diff
@@ -3765,3 +3763,260 @@ def test_schematized_unspecified_service_secretsmanager_v1(self):
 
             assert span.service == DEFAULT_SPAN_SERVICE_NAME
             assert span.name == "aws.secretsmanager.request"
+
+
+    # @pytest.mark.snapshot
```

Comment:
I'd like to add these as snapshot tests, but I can't find documentation/examples of scrubbing out various fields (e.g. dates)

Reply:
Awesome thanks and done